uk.co.compucorp.civicrm.tasksassignments/api/v3/Task.php

```diff
@@ -322,6 +322,7 @@ function civicrm_api3_task_delete($params) {
   }
   
   if (CRM_Activity_BAO_Activity::deleteActivity($params)) {
+    CRM_Tasksassignments_Reminder::sendReminder($params['id']);
```

Comment:
Yes, it might cause some issues. But the sendReminder method needs to find an activity by its ID, so are you sure it even works this way? :O